### PR TITLE
PHP update

### PR DESCRIPTION
**Pull request by rachfop**
## What does this PR do?
This adds support for 95% of PHP docs. 
## Notes to reviewers
Missing the following files:
* how-to-define-workflow-parameters
* how-to-customize-workflow-type
* how-to-define-activity-return-values
* how-to-customize-activity-type
* how-to-register-types-with-a-worker
* how-to-handle-a-signal-in-a-workflow
* how-to-send-a-signal-from-a-client
* how-to-send-a-query-to-a-workflow
* how-to-set-workflow-retry-options
* how-to-set-a-parent-close-policy

### CODE REVIEW

File: docs/php/how-to-define-a-query-in-php.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
+---
+id: how-to-define-a-query-in-php
+title: How to define a Query in PHP
+sidebar_label: Define a Query
+---
+
+Workflows can also answer synchronous [Queries](/php/queries) and receive [Signals](/php/signals).
```
Comment (aarohib) [inline docs/php/how-to-define-a-query-in-php.md:7]:
```suggestion
Workflows can answer synchronous [Queries](/php/queries) and receive [Signals](/php/signals).
```

File: docs/php/how-to-define-a-signal-in-php.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
+---
+id: how-to-define-a-signal-in-php
+title: How to define a Signal in PHP
+sidebar_label: Define a Signal
+---
+
+Workflows can also answer synchronous [Queries](/php/queries) and receive [Signals](/php/signals).
```
Comment (aarohib) [inline docs/php/how-to-define-a-signal-in-php.md:7]:
```suggestion
Workflows can answer synchronous [Queries](/php/queries) and receive [Signals](/php/signals).
```

File: docs/php/how-to-handle-a-query-in-a-workflow-in-php.md [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+        }
+
+        $this->currentState = 'done';
+
+        return null;
+    }
+}
+```
+
+You can also issue a query from code using the `QueryWorkflow()` API on a Temporal client object.
```
Comment (aarohib) [inline docs/php/how-to-handle-a-query-in-a-workflow-in-php.md:79]:
```suggestion
You can also issue a Query from code using the `QueryWorkflow()` API on a Temporal Client object.
```
```suggestion
You can also issue a query from code using the `QueryWorkflow()` API on a Temporal client object.
```

File: docs/php/how-to-handle-a-query-in-a-workflow-in-php.md [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+
+Use `WorkflowStub` to Query Workflow instances from your Client code (can be applied to running Workflows as well):
```
Comment (aarohib) [inline docs/php/how-to-handle-a-query-in-a-workflow-in-php.md:81]:
```suggestion
Use `WorkflowStub` to Query Workflow instances from your Client code (can be applied to both running and closed Workflows):
```
```suggestion
Use `WorkflowStub` to Query Workflow instances from your Client code (can be applied to running Workflows as well):
```

File: docs/php/how-to-send-a-signal-with-start-in-php.md [hunk 5]
```diff
@@ -0,0 +1,18 @@
+---
+id: how-to-send-a-signal-with-start-in-php
+title: How to send Signal-With-Start in PHP
+sidebar_label: Signal-With-Start
+---
+
+You may not know if a Workflow is running and can accept a signal. The `WorkflowClient`->`startWithSignal` API allows you to send a Signal to the current Workflow instance if one exists or to create a new run and then send the Signal.
```
Comment (aarohib) [inline docs/php/how-to-send-a-signal-with-start-in-php.md:7]:
```suggestion
In cases where you may not know if a Workflow is running, and want to send a Signal to it, use `startwithSignal`.
If a running Workflow exists, the `startwithSignal` API sends the Signal. 
If there is no running Workflow, the API starts a new Workflow Run and delivers the Signal to it.
```

File: docs/php/how-to-set-a-schedule-to-close-timeout-in-php.md [hunk 6]
```diff
@@ -0,0 +1,16 @@
+---
+id: how-to-set-a-schedule-to-close-timeout-in-php
+title: How to set Schedule-to-Close Timeout in PHP
+sidebar_label: Schedule-to-Start Timeout
+---
+
+Because Activities are reentrant, only a single stub can be used for multiple Activity invocations.
+The follow code creates an Activity with a `ScheduleToCloseTimeout` set to 2 seconds.
```
Comment (aarohib) [inline docs/php/how-to-set-a-schedule-to-close-timeout-in-php.md:8]:
```suggestion
The following code creates an Activity with a `ScheduleToCloseTimeout` set to 2 seconds.
```
```suggestion
The follow code creates an Activity with a `ScheduleToCloseTimeout` set to 2 seconds.
```

File: docs/php/how-to-set-a-schedule-to-start-timeout-in-php.md [hunk 7]
```diff
@@ -0,0 +1,17 @@
+---
+id: how-to-set-a-schedule-to-start-timeout-in-php
+title: How to set Schedule-to-Start Timeout in PHP
+sidebar_label: Schedule-to-Start Timeout
+---
+
+Because Activities are reentrant, only a single stub can be used for multiple Activity invocations.
+The follow code creates an Activity with a `ScheduleToStartTimeout` set to 10 seconds.
```
Comment (aarohib) [inline docs/php/how-to-set-a-schedule-to-start-timeout-in-php.md:8]:
```suggestion
The following code creates an Activity with a `ScheduleToStartTimeout` set to 10 seconds.
```

File: docs/php/how-to-set-a-start-to-close-timeout-in-php.md [hunk 8]
```diff
@@ -0,0 +1,15 @@
+---
+id: how-to-set-a-start-to-close-timeout-in-php
+title: How to set Start-to-Close Timeout in PHP
+sidebar_label: Start-to-Close Timeout
+---
+
+Because Activities are reentrant, only a single stub can be used for multiple Activity invocations.
+The follow code creates an Activity with a `ScheduleToStartTimeout` set to 2 seconds.
```
Comment (aarohib) [inline docs/php/how-to-set-a-start-to-close-timeout-in-php.md:8]:
```suggestion
The following code creates an Activity with a `StartToCloseTimeout` set to 2 seconds.
```

File: docs/php/how-to-set-a-workflow-run-timeout-in-php.md [hunk 9]
```diff
@@ -0,0 +1,18 @@
+---
+id: how-to-set-a-workflow-run-timeout-in-php
+title: How to set Workflow Run Timeout in PHP
+sidebar_label: Workflow Run Timeout
+---
+
+`workflowRuntTimeout` runs timeout limits duration of a single Workflow invocation.
```
Comment (aarohib) [inline docs/php/how-to-set-a-workflow-run-timeout-in-php.md:7]: Check the init caps on these; in the example code it shows `WorkflowRunTimeout`.

**Review by djmagee — APPROVED** [review]

**Pull request merged**